Gemfile.lock

```diff
@@ -1,38 +1,38 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.0)
-      actionpack (= 6.0.0)
+    actioncable (6.0.2.1)
+      actionpack (= 6.0.2.1)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.0)
-      actionpack (= 6.0.0)
-      activejob (= 6.0.0)
-      activerecord (= 6.0.0)
-      activestorage (= 6.0.0)
-      activesupport (= 6.0.0)
+    actionmailbox (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      activejob (= 6.0.2.1)
+      activerecord (= 6.0.2.1)
+      activestorage (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
       mail (>= 2.7.1)
-    actionmailer (6.0.0)
-      actionpack (= 6.0.0)
-      actionview (= 6.0.0)
-      activejob (= 6.0.0)
+    actionmailer (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      actionview (= 6.0.2.1)
+      activejob (= 6.0.2.1)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.0)
-      actionpack (= 6.0.0)
-      activerecord (= 6.0.0)
-      activestorage (= 6.0.0)
-      activesupport (= 6.0.0)
+    actiontext (6.0.2.1)
```

Comment:
### [actiontext](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.0.0 → 6.0.2.1

[_[change-log](https://github.com/rails/rails/blob/v6.0.2.1/actiontext/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.2.1/actiontext)_]





<details>
<summary>Commits</summary>

A change of **173** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.0...v6.0.2.1).

These are the first 10 commits:
- (f17d964) [Support Rails with sass-rails 6](https://github.com/rails/rails/commit/f17d96475c349e3af7694015f3cc845c4fd7b297)
- (4c8c8cd) [Upgrade the maintainance policy](https://github.com/rails/rails/commit/4c8c8cd8b4c692a3186a78e1e587f416820d1b1d)
- (ebbef80) [Fix missing backtick in maintenance policy version [ci skip]](https://github.com/rails/rails/commit/ebbef80c59a459e1c9ae6f502fbf7950829a8710)
- (26aabdc) [Merge pull request #36969 from AhmedKamal20/patch-1](https://github.com/rails/rails/commit/26aabdcce88e952c9189766bae3d32d5c6448963)
- (e7bd651) [Merge pull request #36981 from tsuka/fix-tag-builder](https://github.com/rails/rails/commit/e7bd651789e01b0f78e83a6e2896821cbb146459)
- (ab91651) [Mark Rails 6.0 released notes as done [ci skip] (#36968)](https://github.com/rails/rails/commit/ab91651a54a843570a219f79c90cdd1a62231d8c)
- (55d8460) [Keep the name of the releases note consistent](https://github.com/rails/rails/commit/55d84602f9704c45bd52a5258628be9542fe8343)
- (70e6ff2) [Generate keep file in tmp/pids](https://github.com/rails/rails/commit/70e6ff2ed8596eb9bd6dcce87cb3f14015638d70)
- (3fda3f9) [Merge pull request #36985 from anmolarora/fix-take-memoizati…](https://github.com/rails/rails/commit/3fda3f9d33875e30dceb58841ee02a36a82fcaed)
- (e8257e3) [Merge pull request #36998 from jhawthorn/reaper_fork](https://github.com/rails/rails/commit/e8257e396c126c7d4012096b28fe3f280271b85f)


</details>